pyalgotrading/algobulls/api.py

```diff
@@ -285,7 +291,7 @@ def set_strategy_config(self, strategy_code: str, strategy_config: dict, trading
         print('Success.')
         return key, response
 
-    def start_strategy_algotrading(self, strategy_code: str, start_timestamp: dt, end_timestamp: dt, trading_type: TradingType, lots: int) -> dict:
+    def start_strategy_algotrading(self, strategy_code: str, start_timestamp: dt, end_timestamp: dt, trading_type: TradingType, lots: int, initial_funds_virtual=1e9, broker_id: int = None, broker_details: dict = None) -> dict:
```

Comment:
`broker_id` parameter not required as everything is coming through `broking_details`. Please clean everywhere.